Apps/WebClient/src/NewClientApp/src/views/LoginView.vue

```diff
@@ -0,0 +1,146 @@
+    return authStore.signIn(redirectPath, idpHint);
+}
+
+function redirect(): void {
+    if (userStore.userIsRegistered) {
+        router.push({ path: defaultPath.value });
+    } else {
+        router.push({ path: "/registration" });
+    }
+}
+
+function signInAndRedirect(idpHint: string): void {
+    signIn(defaultPath.value, idpHint).then(() => {
+        // if this code is reached, the user was already signed in
+        redirect();
+    });
+}
+
+if (oidcIsAuthenticated.value) {
+    redirect();
+} else if (identityProviders.value.length === 1) {
+    signInAndRedirect(identityProviders.value[0].hint);
+} else {
+    isLoading.value = false;
+}
+</script>
+
+<template>
+    <v-container>
+        <LoadingComponent :is-loading="isLoading" />
+        <v-row align="center" justify="center" class="mb-5">
+            <v-col md="5">
+                <v-alert
+                    v-model="showError"
+                    closable
+                    close-label="Close"
+                    type="error"
+                    title="Error"
+                    text="An unexpected error occured while processing the request, please try again."
+                ></v-alert>
+            </v-col>
+        </v-row>
+        <v-row>
+            <v-col>
+                <v-card
+                    v-if="identityProviders && identityProviders.length > 0"
+                    id="loginPicker"
+                    class="shadow-lg bg-white mx-auto"
+                    style="max-width: 25rem"
+                    elevation="7"
+                >
+                    <v-card-title>
+                        <h2 class="text-h5 font-weight-bold pt-3 text-center">
+                            Log In
+                        </h2>
+                    </v-card-title>
+                    <v-card-text
+                        v-if="hasMultipleProviders || isRetry"
+                        class="px-10 py-5"
+                    >
+                        <div
+                            v-for="provider in identityProviders"
+                            :key="provider.id"
+                        >
+                            <HgButtonComponent
+                                :id="`${provider.id}Btn`"
+                                :data-testid="`${provider.id}Btn`"
+                                :disabled="provider.disabled"
+                                variant="primary"
+                                block
+                                class="justify-start"
+                                @click="signInAndRedirect(provider.hint)"
+                            >
+                                <v-icon
+                                    :icon="`${provider.icon}`"
+                                    size="large"
+                                    class="mr-5"
+                                />
+                                <span>{{ provider.name }}</span>
+                            </HgButtonComponent>
```

Comment:
Suggest replacing with `prepend-icon` and `text` props for simplicity.  (This UI is only seen on dev/test.)